bns-test/test/name_transfer.test.ts

```diff
@@ -280,7 +280,7 @@ describe("BNS Test Suite - NAME_TRANSFER", () => {
       "charlie", {
         sender: cases[0].nameOwner
       });
-    expect(receipt.result).include('0x00');
+    expect(receipt.result).include('0x');
```

Comment:
Would this result pass for any zonefile hash?

Reply:
Assertion rewritten in https://github.com/blockstack/stacks-blockchain/pull/2204/commits/0dee5f178e09da48790e901ab1c903c98187ccc5